content/tutorials/manage-the-rippled-server/install-rippled.md

```diff
@@ -46,59 +85,234 @@ This section assumes that you are using CentOS 7 or Red Hat Enterprise Linux 7.
+```
+
+For next steps, see [Postinstall](#postinstall) and [Additional Configuration](#additional-configuration).
 
-        $ sudo systemctl start rippled.service
 
 ## Postinstall
 
-It can take several minutes for `rippled` to sync with the rest of the network, during which time it outputs warnings about missing ledgers. After that, you have a fully functional stock `rippled` server that you can use for local signing and API access to the XRP Ledger.
+It can take several minutes for `rippled` to sync with the rest of the network, during which time it outputs warnings about missing ledgers.
+
+<!--{# ***TODO: for the future, provide a reference for what the most common and unintuitive messages mean -- esp when signaling possible errors.*** #}-->
+
+Once your `rippled` has synchronized with the rest of the network, you have a fully functional stock `rippled` server that you can use for local signing and API access to the XRP Ledger. Use [`rippled` server states](rippled-server-states.html) to tell whether your `rippled` server has synchronized with the network.
+
+For information about communicating with your `rippled` server using the rippled API, see the [rippled API reference](rippled-api.html).
+
+Once you have your stock `rippled` server running, you may want to consider running it as a validating server. For information about validating servers and why you might want to run one, see [Run rippled as a Validator](run-rippled-as-a-validator.html).
+
+
+## Additional Configuration
+
+<!--{# TODO: Once post-rippled-install.md PR is merged, include it here to get latest, consistent info. #}-->
+
+`rippled` should connect to the XRP Ledger with the default configurations. However, you can change your settings by editing the `rippled.cfg` file. For recommendations about configuration settings, see [Capacity Planning](capacity-planning.html).
+
+Changes to the `[node_db]`, `[debug_logfile]`, or `[database_path]` sections may require you to give the `rippled` user and group ownership to your new configured path: ***TODO: I added [node_db] to this list - okay?***
+
+    $ chown -R rippled:rippled <configured path> ***TODO: Is `rippled:rippled` just an example value? When I run this command on macOS, I get `chown: rippled: illegal group name.` When I run it on centos, I get chown: changing ownership of ... : Operation not permitted***
+
+Restart `rippled` for any configuration changes to take effect: ***TODO: when I try this on macOS, I get `sudo: service: command not found`. I don't think the `service` command exists for macOS. For macOS, can the user just stop (ctrl+c? Not sure if that is the proper way to stop the service) and start rippled (sudo ./rippled) instead? For centos and ubuntu, we use `sudo systemctl start rippled.service` to start the service - below should we use `sudo systemctl restart rippled.service` - just to be consistent?***
```

Comment:
The `service` command is part of systemd, which is not part of macOS. Also, running `./rippled` or `sudo ./rippled` is not running it as a service.

So yeah, Ctrl-C would be one way to stop it if you did that. 

 I'm not sure what the macOS equivalent of "setting up a systemd unit to run it as a service" is.

Reply:
Great to know that issuing the `./rippled` command is not running `rippled` as a service. 😅 For now, I'll go with Ctrl-C to stop and `./rippled` to start it up again. For the future, I will look into a command for starting `rippled` as a service for macOS.